browsers/edge.json

```diff
@@ -199,50 +199,57 @@
         },
         "100": {
           "release_date": "2022-04-01",
-          "release_notes": "https://docs.microsoft.com/en-us/deployedge/microsoft-edge-relnote-stable-channel#version-1000118529-april-1",
+          "release_notes": "https://docs.microsoft.com/en-us/deployedge/microsoft-edge-relnote-archive-stable-channel#version-1000118529-april-1",
           "status": "retired",
           "engine": "Blink",
           "engine_version": "100"
         },
         "101": {
           "release_date": "2022-04-29",
-          "release_notes": "https://docs.microsoft.com/en-us/deployedge/microsoft-edge-relnote-stable-channel#version-1010121032-april-28",
+          "release_notes": "https://docs.microsoft.com/en-us/deployedge/microsoft-edge-relnote-archive-stable-channel#version-1010121032-april-28",
           "status": "retired",
           "engine": "Blink",
           "engine_version": "101"
         },
         "102": {
           "release_date": "2022-05-31",
+          "release_notes": "https://docs.microsoft.com/en-us/deployedge/microsoft-edge-relnote-archive-stable-channel#version-1020124530-may-31-2022",
           "status": "retired",
           "engine": "Blink",
-          "engine_version": "102",
-          "release_notes": "https://docs.microsoft.com/en-us/deployedge/microsoft-edge-relnote-stable-channel#version-1020124530-may-31"
+          "engine_version": "102"
         },
         "103": {
-          "release_date": "2022-06-23",
+          "release_date": "2022-07-01",
+          "release_notes": "https://docs.microsoft.com/en-us/deployedge/microsoft-edge-relnote-stable-channel#version-1030126437-june-23",
           "status": "retired",
           "engine": "Blink",
-          "engine_version": "103",
-          "release_notes": "https://docs.microsoft.com/en-us/deployedge/microsoft-edge-relnote-stable-channel#version-1030126437-june-23"
+          "engine_version": "103"
         },
         "104": {
-          "release_date": "2022-08-05",
-          "status": "current",
+          "release_date": "2022-08-09",
```

Comment:
This data was changed a little bit.
I have access to these release dates from our internal releasing tracking system at Microsoft. I know it doesn't match the date on the release notes, but that's the date we have in the system, so I think it's more correct than the notes.